src/AstArgument.h

```diff
@@ -168,22 +152,36 @@ class AstStringConstant : public AstConstant {
 template <typename numericType>  // numericType ⲉ {RamSigned, RamUnsigned, RamFloat}
```

Comment:
Use a capital for template types
```suggestion
// numericType ⲉ {RamSigned, RamUnsigned, RamFloat}
template <typename NumericType> 
```